worldedit-core/src/main/java/com/fastasyncworldedit/core/queue/implementation/blocks/CharSetBlocks.java

```diff
@@ -330,9 +324,19 @@ public boolean hasBiomes(int layer) {
     }
 
     @Override
-    public char[] load(final int layer) {
+    public char[] loadChars(final int layer) {
         updateSectionIndexRange(layer);
-        return super.load(layer);
+        return super.loadChars(layer);
+    }
+
+    @Override
+    public int[] loadInts(final int layer) {
+        return null;
```

Comment:
Rather than returning null, default methods in the IChunkSet interface could return converted arrays between char and int. OR, throw a exception detailing that the class is not for int usage and something's gone very wrong

Reply:
I went ahead and just had them throw exceptions with a message to report it to the FastAsyncWorldEdit developers.